edx_notifications/server/web/templates/django/notifications_widget.html

```diff
@@ -0,0 +1,31 @@
+{% comment %}
+
+This is a "drop-in" Django template widget to instantiate the edx-notifications front end application
+which is a Backbone application that is defined in edx_notifications/server/web/static/js
+
+This HTML snippet assumes that jQuery been included in the page that is using this widget
+
+{% endcomment %}
+
+<div id='edx_notification_container'></div>
+<div id="edx_notification_pane"></div>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/underscore.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/vendor/backbone.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/models/counter_icon_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/user_notification_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/notification_collection_model.js"></script>
+
+<script src="{{ STATIC_URL }}edx_notifications/js/views/counter_icon_view.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/views/notification_pane_view.js"></script>
+
+
+<script type="text/javascript">
+    /* start up the root Notification view, which will bootstrap the whole front-end */
+    var counter_icon = new CounterIconView({
+        el: $("#edx_notification_container"),
+        model: new CounterIconModel(),
+        pane_el: $("#edx_notification_pane")
```

Comment:
yea, let me sit down with you with the mockups. The notification icon will be a permanent element in a header region and then the pane is an overlay which will appear over the rest of the page when the notification icon is clicked. I wasn't sure whether it'd be more easy to get the overlay behavior by having it as a separate element or not.